src/sphinx/archetypes/java_app/index.rst

```diff
@@ -147,6 +147,67 @@ application arguments with ``--``. For example
     ./bin/my-app -Dconfig.resource=prod.conf -- -appParam1 -appParam2
 
 
+Multiple Applications
+=====================
+
+If you have multiple main classes then the ``JavaAppPackaging`` archetype provides you with two different ways of
+generating start scripts.
+
+1. A start script for each entry point. This is the default behaviour, when no ``mainClass in Compile`` is set
+2. One start script for the defined ``mainClass in Compile`` and forwarding scripts for all other main classes.
+
```

Comment:
True. I'm still not happy with the naming anyway. _Forwarder_, _Delegate_,..
However I will define the semantics more clearly.